lightning/src/ln/msgs.rs

```diff
@@ -543,6 +543,8 @@ pub struct TxSignatures {
 	pub tx_hash: Txid,
 	/// The list of witnesses
 	pub witnesses: Vec<Witness>,
+	/// Optional signature for shared funding outpoint input (signed by both parties)
+	pub tlvs: Option<Signature>,
```

Comment:
We must refer to the actual TLV here and below.

```suggestion
	/// Optional signature for shared funding outpoint input (signed by both parties)
	pub funding_outpoint_sig: Option<Signature>,
```

Reply:
In the spec the field is named `tlvs`, and while I think this is not particularly a descriptive name, I think it makes sense to stick to the spec regarding message names and message field names. What do you think?
Also, the spec talks about shared signatures, which is a bit more general than funding outpoint signatures, so we should not restrict it (it is currently used for funding outputs only, but I suppose the intent is to keep the naming more generic).

Reply:
> In the spec the field is named `tlvs`, and while I think this is not particularly a descriptive name, I think it makes sense to stick to the spec regarding message names and message field names. 

So I think there is just a bit of confusion regarding the TLV stream for messages here (the spec is a bit weird in how it lays things out for messages ) but `tlvs` is really just describing the TLV stream that comes at the end of most messages. You'd need to name the individual TLV fields on our side. `tlvs` is not a `Signature` but the 0th tlvs field (`funding_outpoint_sig`) is. The `tlvs` field is just there for convenience to later describe the tlv stream fields in `txsig_tlvs` in the spec.